### PR TITLE
Fixed bug when energy site name is null

### PR DESCRIPTION
**Pull request by giachello**
closes https://github.com/alandtse/tesla/issues/62

### CODE REVIEW

File: teslajsonpy/controller.py [hunk 1]
```diff
@@ -346,7 +346,10 @@ async def connect(
             energysite_id = energysite["energy_site_id"]
             self.__id_energysiteid_map[energysite["id"]] = energysite_id
             self.__energysiteid_id_map[energysite_id] = energysite["id"]
-            self.__energysite_name[energysite_id] = energysite["site_name"]
+            if "site_name" in energysite:
+                self.__energysite_name[energysite_id] = energysite["site_name"]
```
Comment (alandtse) [inline teslajsonpy/controller.py:350]:
```python
                self.__energysite_name[energysite_id] = energysite.get("site_name", "My Home")
```

Comment (alandtse) [reply teslajsonpy/controller.py:350]: Of course, please lint and make sure it works with a test.

**Review by alandtse — CHANGES_REQUESTED** [review]

**Comment by gwendalg** [comment]
The change is not enough: "site_name" is also used in homeassistant/power.py, line 37:
`self._display_name: Text = data["site_name"]`
Using .get() with a default value is preferable, as the `_display_name` field is actively used in homeassistant.

**Comment by giachello** [comment]
Using get and fixed power.py as well. implemented tests.

**Pull request merged**